tasks/quick_mode.yml

```diff
@@ -0,0 +1,23 @@
+- name: list files described as changed by git
+  shell: |
+    set -o pipefail -e
+    git status --short . | grep -v '^\\sD' | awk '{print $2}'
+  args:
+    chdir: "{{ osp_templates_input_dir }}"
+    executable: /bin/bash
+  register: _shell_output
+  changed_when: false
+  delegate_to: localhost
+
+- debug:
```

Comment:
Should this debug statement be here?

Reply:
Since the git command above is a bit esoteric, it helps to have this list of which files are found and are actually going to be templated. I think it makes sense to leave this in place, as it was in the original role.